tools/deb/description-discovery

```diff
@@ -0,0 +1 @@
+Istio Pilot Discovery provides services discovery for the http://istio.io project.
```

Comment:
earnest suggestion: just call it Istio Pilot, to keep in line with terminology used everywhere. The phrase discovery service (coming from envoy) is super confusing and sounds strange. 
Also, Istio Pilot is not service discovery alone. It does a lot more than just plain discovery. May be say, Istio Pilot provides service discovery and traffic control capabilities for microservices in the Istio service mesh